datalad_container/extractors/_load_singularity_versions.py

```diff
@@ -0,0 +1,35 @@
+"""
+Importing this file extends datalad.support.external_version:
+
+Adds:
+    - external_versions["cmd:apptainer"]
+    - external_versions["cmd:singularity"]
+"""
+import subprocess
+
+from datalad.cmd import (
+    StdOutCapture,
+    WitlessRunner,
+)
+from datalad.support.external_versions import external_versions
+
+
+def __get_apptainer_version():
+    version = WitlessRunner().run("apptainer --version", protocol=StdOutCapture)['stdout'].strip()
+    strip_str = len("apptainer version ")
+    return version[strip_str:]
```

Comment:
another pattern for the same thing for your consideration

```suggestion
    return version.split("apptainer version", 1)[1].strip()
```